molecule/default/verify.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Verify
-  hosts: all
+  hosts: nginx
```

Comment:
You are setting target to hosts `nginx` but all tasks use `delegate_to: localhost`.
Why can't we set target hosts to `localhost` directly and remove tasks delegations?